accessibility/.pa11yci-mobile.json

```diff
@@ -0,0 +1,23 @@
+{
+  "defaults": {
+    "timeout": 240000,
+    "runners": ["axe"],
+    "ignore": ["color-contrast"],
+    "concurrency": 1,
+    "chromeLaunchConfig": {
+      "ignoreHTTPSErrors": true,
+      "args": ["--disable-dev-shm-usage", "--no-sandbox"]
+    },
+    "viewport": {
+      "width": 390,
+      "height": 844,
+      "mobile": true
+    },
+    "actions": [
+      "wait for element #main-content to be visible",
+      "screen capture screenshots-output/mobile-main-view.png",
+      "click element .usa-navbar button",
+      "screen capture screenshots-output/mobile-expand-menu.png"
```

Comment:
do we expect these actions to be applicable to all projects or are projects supposed to configure things?

Reply:
We set  `#main-content` in our next template layout - so it may not be available on all projects (same for `usa-navbar` in the header).

A more universally applicable alternative might be these actions:
```
  "wait for element body to be visible",
   "wait for document to be ready",
 ```

But as soon as someone pulls together a platform to start a project - is there already a checklist of things they need to customize? 

Are we trying to do documentation in this PR?

Reply:
waiting for element body to be visible or document ready both sound good to me. defer to you.
ideally i'd prefer something that doesn't fail on the first PR run through.
and yeah let's add some basic documentation. how about a file `docs/accessibility/accessibility-checks.md` that explains:

- high level overview of how accessibility checks works in this repo—uses pa11y tool, runs on PR environments by default (link to the docs on pr-environments), which workflows trigger checks (link to the workflow)
- how to customize (link to the config files and the pa11y documentation)

Reply:
Just sticking with `"wait for element body to be visible",`